docs/WifiPerformance.md

```
@@ -60,980 +60,912 @@ All wireless adapters were tested under consistent conditions - each positioned
-    signal: -37 dBm
-    tx bitrate: 434.0 MBit/s
-    ```
+    RX: 43610 bytes (147 packets)
+    TX: 47436 bytes (197 packets)
+    signal: -40 dBm
+    rx bitrate: 1200.9 MBit/s 80MHz HE-MCS 11 HE-NSS 2 HE-GI 0 HE-DCM 0
+    tx bitrate: 1200.9 MBit/s 80MHz HE-MCS 11 HE-NSS 2 HE-GI 0 HE-DCM 0
+```
 
-### Atheros AR9271
+### N
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)